packages/vite-plugin-nitro/src/lib/options.ts

```diff
@@ -31,3 +33,32 @@ export interface PrerenderOptions {
 export interface SitemapConfig {
   host: string;
 }
+
+export interface PrerenderContentDir {
+  /**
+   * The directory where files should be grabbed from.
+   * @example `/src/contents/blog`
+   */
+  contentDir: string;
+  /**
+   * Transform the matching content files path into a route.
+   * The function is called for each matching content file within the specified contentDir.
+   * @param file information of the matching file (`path`, `name`, `extension`, `frontmatter`)
+   * @returns a string with the route should be returned (e. g. `/blog/<slug>`) or the value `false`, when the route should not be prerendered.
+   */
+  transform: (file: ContentFile) => string | false;
+}
+
+/**
+ * @param path the path to the content file
+ * @param name the basename of the matching content file without the file extension
+ * @param extension the file extension
+ * @param frontmatter the frontmatter attributes extracted from the frontmatter section of the file
+ * @returns a string with the route should be returned (e. g. `/blog/<slug>`) or the value `false`, when the route should not be prerendered.
+ */
+export interface ContentFile {
+  path: string;
+  frontmatter: Record<string, any>;
```

Comment:
Noticed this on the second pass. There is a similar interface in the @analogjs/content package named `ContentFile`. I think we should rename this to `PrerenderContentFile` and change `frontmatter` to `attributes`.

Reply:
Done :)